library/dd-sdk-android-session-replay/apiSurface

```diff
@@ -1,342 +1,13 @@
-data class com.datadog.android.sessionreplay.model.FullSnapshotRecord
```

Comment:
why all this is gone?

Reply:
because I changed the model generation methods by excluding some schemas that were already referenced inside others.